### PR TITLE
feat(regexp): add a regexp for disk-opts names filter

### LINKED ISSUE #66: Allow regexp to whitelist disks
Hi team,

I saw that regexp whitelisting is available for network interfaces (and that's really cool ! :) ).

Is it possible for you to add the same process for disk ?

Thank you,

### PR DESCRIPTION
**Pull request by aurrelhebert**
fix #66

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -225,6 +225,14 @@ disk-opts:
     - sda3
 ```
 
+Disk-opts, names field support now regular expression to white-list disks names based on [golang MatchString](https://golang.org/pkg/regexp/#MatchString) implementation. However to use a regular expression you need to prefix the string value by a `~`. To whitelist all eth interfaces, you can set:
```
Comment (miton18) [inline Readme.md]: To whitelist all eth interfaces, => disk

Comment (miton18) [inline Readme.md]: You can use a more standard version of regex with https://golang.org/pkg/regexp/#CompilePOSIX, with it, it won't be related with Go impl

File: collectors/disk.go [hunk 2]
```diff
@@ -104,7 +106,7 @@ func (c *Disk) scrape() error {
 	for diskPath, usage := range dev {
 		if len(c.allowedDisks) > 0 {
 			_, diskName := path.Split(diskPath) // return "sda1" from "/dev/sda1"
-			if _, allowed := c.allowedDisks[diskName]; !allowed {
+			if !stringInSlice(diskName, c.allowedDisks) {
```
Comment (miton18) [inline collectors/disk.go]: it's just a `contains`, not a regex match ?

File: Readme.md (hunk 1 above)
Comment (aurrelhebert) [reply Readme.md]: This RegExp is already used on net iterfaces

File: collectors/disk.go (hunk 2 above)
Comment (aurrelhebert) [reply collectors/disk.go]: Check the stringInSlice implementation ;)

**Pull request merged**